sync-android/src/test/java/com/cloudant/sync/datastore/encryption/TestUtils.java

```diff
@@ -0,0 +1,178 @@
+/**
```

Comment:
Instead of copying this file you should be able to supplement the sourcesets in the gradle file, someting like
```
java {
  sourceSets {
    test {
      // some gradle magic
    }
  }
}